app/themes/user-admin/src/index.html

```diff
@@ -8,9 +8,10 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 
-  <link rel='stylesheet' href='//docker.local:8088/themes/bootstrap/css/styles.min.css' type='text/css' />
+  
   <link rel='stylesheet' href='//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css' type='text/css' />
   <link rel='stylesheet' href='//docker.local:8088/themes/bootstrap/css/font-awesome-animation.css' type='text/css' />
```

Comment:
Please update these CSS references to point to the root domain, not `docker.local`. This will cause problems when we attempt to deploy this project to another domain (eg www.example.com/admin)